templates/redis.conf.epp

```diff
@@ -432,6 +434,37 @@ slave-serve-stale-data <%= bool2str($slave_serve_stale_data, 'yes', 'no') %>
 # administrative / dangerous commands.
 slave-read-only <%= bool2str($slave_read_only, 'yes', 'no') %>
 
+# A Redis master is able to list the address and port of the attached
+# replicas in different ways. For example the "INFO replication" section
+# offers this information, which is used, among other tools, by
+# Redis Sentinel in order to discover replica instances.
+# Another place where this info is available is in the output of the
+# "ROLE" command of a master.
+#
+# The listed IP address and port normally reported by a replica is
+# obtained in the following way:
+#
+#   IP: The address is auto detected by checking the peer address
+#   of the socket used by the replica to connect with the master.
+#
+#   Port: The port is communicated by the replica during the replication
+#   handshake, and is normally the port that the replica is using to
+#   listen for connections.
+#
+# However when port forwarding or Network Address Translation (NAT) is
+# used, the replica may actually be reachable via different IP and port
+# pairs. The following two options can be used by a replica in order to
+# report to its master a specific set of IP and port, so that both INFO
+# and ROLE will report those values.
+#
+# There is no need to use both the options if you need to override just
+# the port or the IP address.
+#
+# replica-announce-ip 5.5.5.5
+# replica-announce-port 1234
+<% if $repl_announce_ip { -%>replica-announce-ip <%= $repl_announce_ip %><% } -%>
+<% if $repl_announce_port { -%>replica-announce-port <%= $repl_announce_port %><% } %>
```

Comment:
Maybe for consistency:

```suggestion
<% if $repl_announce_port { -%>replica-announce-port <%= $repl_announce_port %><% } -%>
```

Reply:
Hello Smortex, you're right, it seems better. I fixed it.